commercial-schedules/src/main/java/org/dcsa/conformance/standards/cs/CsScenarioListBuilder.java

```diff
@@ -10,175 +10,380 @@
+            Map.entry(
+                "Vessel Schedules - Service",
+                noAction()
+                    .thenEither(
+                        scenarioWithParametersVs(CARRIER_SERVICE_CODE),
+                        scenarioWithParametersVs(UNIVERSAL_SERVICE_REFERENCE))),
+            Map.entry(
+                "Vessel Schedules - Vessel",
+                noAction()
+                    .thenEither(
+                        scenarioWithParametersVs(VESSEL_IMO_NUMBER),
+                        scenarioWithParametersVs(VESSEL_IMO_NUMBER, CARRIER_SERVICE_CODE),
+                        scenarioWithParametersVs(VESSEL_IMO_NUMBER, UNIVERSAL_SERVICE_REFERENCE),
+                        scenarioWithParametersVs(
+                            VESSEL_IMO_NUMBER, CARRIER_SERVICE_CODE, CARRIER_VOYAGE_NUMBER),
+                        scenarioWithParametersVs(
+                            VESSEL_IMO_NUMBER, CARRIER_SERVICE_CODE, UNIVERSAL_VOYAGE_REFERENCE),
+                        scenarioWithParametersVs(
+                            VESSEL_IMO_NUMBER, UNIVERSAL_SERVICE_REFERENCE, CARRIER_VOYAGE_NUMBER),
+                        scenarioWithParametersVs(
+                            VESSEL_IMO_NUMBER,
+                            UNIVERSAL_SERVICE_REFERENCE,
+                            UNIVERSAL_VOYAGE_REFERENCE))),
+            Map.entry(
+                "Vessel Schedules - Voyage",
+                noAction()
+                    .thenEither(
+                        scenarioWithParametersVs(CARRIER_VOYAGE_NUMBER, CARRIER_SERVICE_CODE),
+                        scenarioWithParametersVs(
+                            CARRIER_VOYAGE_NUMBER, UNIVERSAL_SERVICE_REFERENCE),
+                        scenarioWithParametersVs(UNIVERSAL_VOYAGE_REFERENCE, CARRIER_SERVICE_CODE),
+                        scenarioWithParametersVs(
+                            UNIVERSAL_VOYAGE_REFERENCE, UNIVERSAL_SERVICE_REFERENCE))),
+            Map.entry(
+                "Vessel Schedules - Location",
+                noAction()
+                    .thenEither(
+                        scenarioWithParametersVs(UN_LOCATION_CODE),
+                        scenarioWithParametersVs(UN_LOCATION_CODE, FACILITY_SMDG_CODE))),
+            Map.entry(
+                "Limit and Pagination",
+                noAction()
+                    .thenEither(
+                        scenarioWithParametersPtpForPagination(
+                            getPtpRoutings(), PLACE_OF_RECEIPT, PLACE_OF_DELIVERY, LIMIT),
+                        scenarioWithParametersPsForPagination(
+                            getPortSchedules(), UN_LOCATION_CODE, DATE, LIMIT),
+                        scenarioWithParametersVsForPagination(
+                            getVesselSchedules(), VESSEL_IMO_NUMBER, LIMIT))))
+        .collect(
+            Collectors.toMap(
+                Map.Entry::getKey, Map.Entry::getValue, (e1, e2) -> e1, LinkedHashMap::new));
```

Comment:
Oops.. formatting this code does not make it better 🤔  
Also hard to see if you actually changes something btw.